{"uuid": "ecfce72b-ba3f-4bb4-a8d8-a6da220cf141", "children": ["0c4fc111-3344-4d2b-97dd-6ae03132887d"], "befores": [{"name": "_xunit_setup_method_fixture_TestLogin", "status": "passed", "start": 1721231171923, "stop": 1721231171959}], "afters": [{"name": "_xunit_setup_method_fixture_TestLogin::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"E:\\SoftWare\\Python3.10\\lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"E:\\SoftWare\\Python3.10\\lib\\site-packages\\_pytest\\fixtures.py\", line 896, in _teardown_yield_fixture\n    next(it)\n  File \"E:\\SoftWare\\Python3.10\\lib\\site-packages\\_pytest\\python.py\", line 824, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"E:\\SoftWare\\Python3.10\\lib\\site-packages\\_pytest\\python.py\", line 716, in _call_with_optional_argument\n    func()\n  File \"E:\\project\\PycharmProjects\\HM_TPshop_Test\\web\\scripts\\test_pytest_login.py\", line 38, in teardown_method\n    WebDriverWait(self.driver, 10).until(EC.url_to_be(\"http://192.168.249.129/index.php/Home/User/index.html\"))\n  File \"E:\\SoftWare\\Python3.10\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1721231172040, "stop": 1721231182287}], "start": 1721231171923, "stop": 1721231182287}